Source/WebKit/NetworkProcess/NetworkProcess.cpp

```diff
@@ -1732,6 +1732,7 @@ static Vector<WebCore::SecurityOriginData> filterForRegistrableDomains(const Has
 
 void NetworkProcess::deleteAndRestrictWebsiteDataForRegistrableDomains(PAL::SessionID sessionID, OptionSet<WebsiteDataType> websiteDataTypes, RegistrableDomainsToDeleteOrRestrictWebsiteDataFor&& domains, bool shouldNotifyPage, CompletionHandler<void(HashSet<RegistrableDomain>&&)>&& completionHandler)
 {
+    RELEASE_LOG(Storage, "NetworkProcess::deleteAndRestrictWebsiteDataForRegistrableDomains started to delete and restrict data for session %" PRIu64, sessionID.toUInt64());
```

Comment:
I think we should add info on what _type_ website data is about to be deleted. Not down to each type but based on the ITP enum that carves out script-written website data.

Reply:
I think this function is called with all types that ITP monitors, see `ResourceLoadStatisticsStore::removeDataRecords`.
Do you mean to log the size of members (e.g. `domainsToDeleteAllCookiesFor`, `domainsToDeleteAllButHttpOnlyCookiesFor`, `domainsToDeleteAllScriptWrittenStorageFor`, etc) of `RegistrableDomainsToDeleteOrRestrictWebsiteDataFor domains` here?